### ISSUE TITLE
remove storeInstance.__internal__ from getters ?

### ISSUE THREAD

**Issue by alex-drocks** [sindresorhus/electron-store#151]
Hello @sindresorhus your electron-store library is awesome.

What do you think of the possibility of removing / filtering-out the **__internal__** property when getting the instance.store data?

I'm talking about this auto-generated property:
```JavaScript
"__internal__": {
    "migrations": {
      "version": "0.9.2"
    }
},
```

Use case:
```JavaScript
            migrations: {
                '>=0.9.2': clientStore => {
                    const clients = Object.values(clientStore.store).filter(value => value.nom);
                    clients.forEach(client => {
                        client.id = client.id ? client.id : uuidv4();
                        clientStore.set(client.nom, client);
                    });
                },
            }
```

As the example suggests, you get the complete file output when getting the store data so you have to remember filtering out the __internal__ property to avoid unhandled error when accessing the objects. In the example I validate that the value has a .nom property to make sure I'm not trying to do a migration on the __internal__ property.

Could there be something similar to this cleanup function directly in the getters of your library ? Like at the source where you populate the returned data ?
```JavaScript
    loadFile(storeInstance) {
        const fileData = storeInstance.store; // reads the json file and returns it as a JS object copy
        delete fileData['__internal__']; //remove useless internal property from the object copy
        return fileData; // return clean object of file data
    },

```

I hope I make sense, please let me know what you think of that.

Thanks,
Alexandre Desroches

**Comment by sindresorhus** [comment]
Yes, that should be done. It's just an oversight that it's available in the `.store` property.

**Comment by alex-drocks** [comment]
@sindresorhus Nice, thank you!

**Comment by sindresorhus** [comment]
`.store` should not expose `__internal__`. `.size` and iteration already skip it, so this is an oversight rather than a design decision.

It is a `conf` issue and the fix is being made there.
